packages/derived-wallet-base/src/abstraction.ts

```diff
@@ -1,43 +1,15 @@
-import { AccountAddress, AuthenticationKey, hashValues, isValidFunctionInfo, Serializer } from '@aptos-labs/ts-sdk';
+import { AuthenticationKey, DerivableAbstractedAccount } from '@aptos-labs/ts-sdk';
 
 /**
- * The domain separator used to calculate the DAA account address.
- */
-export const ADDRESS_DOMAIN_SEPARATOR = 5;
-
-/**
- * Compute the account address of the DAA
- * The DAA account address is computed by hashing the function info and the account identity
- * and appending the domain separator (5)
- *
  * @param functionInfo - The authentication function
  * @param accountIdentifier - The account identity
  * @returns The account address
  */
-export function computeDomainAuthenticationKey(functionInfo: string, accountIdentifier: Uint8Array) {
-  if (!isValidFunctionInfo(functionInfo)) {
-    throw new Error(`Invalid authentication function ${functionInfo}`);
+export function computeDomainAuthenticationKey(functionInfo: string, accountIdentifier: Uint8Array): AuthenticationKey {
+  try{
+    const add = DerivableAbstractedAccount.computeAccountAddress(functionInfo, accountIdentifier);
+    return new AuthenticationKey({ data:add });
```

Comment:
Unrelated to this PR: I would suggest possibly renaming the original function from the SDK.
I think "authentication key" would be more apt.

For context, the address of an account always matches the authentcation key of its original public key.
It only differs when the authentication key is rotated.

Reply:
tbh, authentication key is a bit confusing. I understand the technical side and why it makes sense but I am not completely sure devs really understand what authentication key is for and its meaning.
IMO, account address is a more understandable and used term. I would actually say we should rename it in the wallet adapter.

Also, we might in the future encourage wallets to rotate their auth key to be able to use it in a native Aptos wallet.